src/components/App/App.tsx

```diff
@@ -1383,44 +1416,26 @@ export default class App extends React.Component<null, AppState> {
                       ></Progress>
                     </div>
                   )}
-                  {this.state.state === 'connected' && (
-                    <VideoChat
-                      socket={this.socket}
-                      participants={this.state.participants}
-                      nameMap={this.state.nameMap}
-                      pictureMap={this.state.pictureMap}
-                      tsMap={this.state.tsMap}
-                      rosterUpdateTS={this.state.rosterUpdateTS}
-                    />
-                  )}
+                  <Footer>
+                    {this.state.state === 'connected' && (
+                      <VideoChat
+                        socket={this.socket}
+                        participants={this.state.participants}
+                        nameMap={this.state.nameMap}
+                        pictureMap={this.state.pictureMap}
+                        tsMap={this.state.tsMap}
+                        rosterUpdateTS={this.state.rosterUpdateTS}
+                      />
+                    )}
+                  </Footer>
                 </div>
-              </Grid.Column>
-              <Grid.Column
-                width={4}
-                style={{ display: 'flex', flexDirection: 'column' }}
-                className="fullHeightColumn"
-              >
-                <Input
-                  inverted
-                  fluid
-                  label={'My name is:'}
-                  value={this.state.myName}
-                  onChange={this.updateName}
-                  icon={
-                    <Icon
-                      onClick={() =>
-                        this.updateName(null, { value: generateName() })
-                      }
-                      name="refresh"
-                      inverted
-                      circular
-                      link
-                    />
-                  }
-                />
-                {/* <Divider inverted horizontal></Divider> */}
+              </div>
+              <PlaylistSidebar />
```

Comment:
Wondering if it's easier to just implement the playlist as a modal or popover rather than restructuring the App so much

Reply:
Sure instead of using a sidebar we can add a modal for that.